hot/advanced.yaml

```diff
@@ -70,7 +87,10 @@ resources:
         package_upgrade: false
         packages: { get_param: packages }
         runcmd:
-          - { get_attr: ['all_done_handle', 'curl_cli'] }
+          - str_replace:
+              template: "%curl --verbose --retry 7"
+              params:
+                '%curl': { get_attr: ['all_done_handle', 'curl_cli'] }
```

Comment:
Let's drop this change. Now that we have IPv6 support in the template, the `curl` invocation should just work, and we don't need to complicate things by adding a `str_replace` invocation.